flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg

```diff
@@ -17,10 +17,11 @@ major = 1
 minor = 0
 
 [WEIGHT_AND_BALANCE]
-max_gross_weight = 174165 ; Empty weight, (LBS)
-empty_weight = 90400 ; Empty weight, (LBS)
+max_gross_weight = 174165 ; Maximum total weight of the aircraft when fully loaded, (LBS)
```

Comment:
Max gross weight comment should be "Maximum takeoff weight" rather than "Maximum total weigh of the aircraft when fully loaded." At least, that is the way this variable appears to be used by MSFS. Ramp weight can be higher.

Reply:
Fixed ✅

Reply:
Looks good

Reply:
If we could change the MSFS weight and balance UI to say Max Ramp Weight instead of Max Takeoff Weight, we could put the "Max ramp weight (maximum total weight of the airplane when fully loaded" back and a max gross weight value of 175045 in.

Reply:
> If we could change the MSFS weight and balance UI to say Max Ramp Weight instead of Max Takeoff Weight, we could put the "Max ramp weight (maximum total weight of the airplane when fully loaded" back and a max gross weight value of 175045 in.

I don't think that is modifiable. We can only edit the names of the stations as of now